modules/metrics/mule-metrics-exporter-impl/pom.xml

```diff
@@ -15,11 +15,24 @@
     <description>Definitions for Mule Metrics Exporter Implementation</description>
     
     <properties>
+        <muleBomVersion>4.7.0-SNAPSHOT</muleBomVersion>
         <formatterConfigPath>../../../formatter.xml</formatterConfigPath>
         <javaModuleName>org.mule.runtime.metrics.exporter.impl</javaModuleName>
         <ioGrpcVersion>1.51.0</ioGrpcVersion>
     </properties>
-    
+
```

Comment:
Is this really needed? Dows it not get propagated from the parent project?

Reply:
You are right. I will update the PR.